### PR TITLE
Replaced all /en links

### LINKED ISSUE #3475: Replace all /en links
Find all /en links in the repo and remove the locale part of the link

### PR DESCRIPTION
**Pull request by Patil2099**
Closes #3475
Removes all /en links except en-US

### CODE REVIEW

File: README-FOR-WAGTAIL-LOCALISATION.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The modeltranslation library adds localised URLs to Wagtail, interjecting the lo
 
 with "en" as locale, becomes:
 
-    https://foundation.mozilla.org/en/campaigns/aadhaar
+    https://foundation.mozilla.org/campaigns/aadhaar
```
Comment (Pomax) [inline README-FOR-WAGTAIL-LOCALISATION.md]: this one needs to stay, as the text literally explains that hitting a URL without a locale will inject one =)

**Comment by Pomax** [comment]
Thanks so much for helping out @Patil2099! just a small issue to fix and this can get landed =)

**Comment by Patil2099** [comment]
@Pomax I have applied the requested change.

**Comment by Patil2099** [comment]
@Pomax Is there something wrong with my PR?

**Pull request closed without merging**